src/CMakeLists.txt

```diff
@@ -109,6 +109,7 @@ set(SRC
     stdlib_hashmaps.f90
     stdlib_hashmap_chaining.f90
     stdlib_hashmap_open.f90
+    stdlib_io_filesystem.f90
```

Comment:
```suggestion
    stdlib_io_filesystem.F90
```

If you capitalise the extension compilers will turn on their respective preprocessors

Reply:
It worked, thanks 👍